src/client/scripts/exportRoom.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+/* global Cards:false Rooms:true*/
+
+Template.exportRoom.helpers({
+
+    getCards : function(category) {
+        var mongoQuery = {
+            "roomCode": Session.get("roomNumber"),
+            "category": category
+        };
+
+        return Meteor.apply("getCardCollectionByQuery",
```

Comment:
I don't think this matters for templates though, it will render with the undefined, then just update once the data returns. Blaze is smart yo. Feel free to tell if I'm wrong though and it doesn't work.